src/main/java/org/opentripplanner/routing/api/request/PassThroughPoint.java

```diff
@@ -0,0 +1,36 @@
+package org.opentripplanner.routing.api.request;
+
+import java.util.List;
+import org.opentripplanner.transit.model.site.StopLocation;
+
+/**
+ * Defines one pass-through point which the journey must pass by.
+ */
+public class PassThroughPoint {
```

Comment:
Is there a reason this is not a record? There is not encapsulation happening so might as well be one, isn't it?

Reply:
I agree, was not sure on policy.